cluster/gce/addons/loadbalancing/cloud-provider-binding.yaml

```diff
@@ -3,6 +3,38 @@ kind: RoleBinding
 metadata:
   labels:
     addonmanager.kubernetes.io/mode: Reconcile
+  name: gce:cloud-provider
+  namespace: kube-system
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: gce:cloud-provider
+subjects:
+- kind: ServiceAccount
+  name: cloud-provider
+  namespace: kube-system
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  labels:
+    addonmanager.kubernetes.io/mode: Reconcile
+  name: gce:cloud-provider
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: gce:cloud-provider
+subjects:
+- kind: ServiceAccount
+  name: cloud-provider
+  namespace: kube-system
+---
+# DEPRECATED: cloud-provider roles and bindings are deprecated. Use gce:cloud-provider instead.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
```

Comment:
We discussed putting the deprecation notice in an annotation, and keeping Reconcile so the annotation is populated.

Reply:
Sorry Tim I can't find an example annotation to denote deprecation. But I did find an related (closed without fixes) issue: https://github.com/kubernetes/kubernetes/issues/38269.